### PR TITLE
Add bigquery to source_defintions yaml

### PR DESCRIPTION
**Pull request by sherifnada**

### CODE REVIEW

File: airbyte-config/init/src/main/resources/seed/source_definitions.yaml [hunk 1]
```diff
@@ -394,3 +394,8 @@
   dockerRepository: airbyte/source-typeform
   dockerImageTag: 0.1.0
   documentationUrl: https://docs.airbyte.io/integrations/sources/typeform
+- sourceDefinitionId: bfd1ddf8-ae8a-4620-b1d7-55597d2ba08c
+  name: BigQuery
+  dockerRepository: airbyte/source-bigquery
+  dockerImageTag: 0.1.0
+  documentationUrl: https://docs.airbyte.io/integrations/sources/bigquery
```
Comment (jrhizor) [inline airbyte-config/init/src/main/resources/seed/source_definitions.yaml:401]: Is this pr blocked until the docs are added here?

Comment (jrhizor) [reply airbyte-config/init/src/main/resources/seed/source_definitions.yaml:401]: Also, this is the "correct" link once it's created but `airbyte-integrations/connectors/source-bigquery/src/main/resources/spec.json` points to `/source/bigquery` instead.

Comment (sherifnada) [reply airbyte-config/init/src/main/resources/seed/source_definitions.yaml:401]: yes, @heade is working on finishing docs

**Comment by DoNotPanicUA** [comment]
Included the change into the PR #4981

**Pull request closed without merging**